keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -117,6 +118,25 @@ class SqlVerificationRepository(
     }
   }
 
+  override fun wasSuccessfullyVerifiedIn(
+    deliveryConfig: DeliveryConfig,
+    artifactReference: String,
+    version: String,
+    environmentName: String,
+    verification : Verification
+  ) =
+    getState(
+      VerificationContext(
+        deliveryConfig,
+        environmentName,
+        artifactReference,
+        version
+      ),
```

Comment:
Side-note: I just realized this is precisely what a `Deployment` class might model if we were to introduce that. It would make a bunch of our method signatures shorter and more self-explanatory (like in this case: "was deployment X successfully verified with verification Y?").
Cc: @robfletcher

Reply:
P.S. @robfletcher Any chance you might agree with just renaming that class?

Reply:
Yeah, if we are considering using it in a wider context